.github/workflows/ci.yml

```diff
@@ -233,7 +233,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-12, windows-2019, windows-2022]
-        go-version: ["1.17.13", "1.19.9"]
+        go-version: ["1.17.13", "1.19.10"]
```

Comment:
Why is there still go1.17 version? 

Shouldnt we be ideally building for go1.20 and go1.19?

There is https://github.com/containerd/containerd/pull/8103 which is marked to be picked to `release/1.6`, but was not done.

Reply:
1.17 was the oldest version that was supported when the 1.6 LTS branch was started, so this is mostly to verify the "lower limit" doesn't break. I think it's mostly considered a "best effort", but trying to keep compatibility with 1.17 as long as reasonably possible.

Reply:
Should we add 1.20 also to the matrix ?

Reply:
Given that 1.6 is the LTS branch, we'll probably have to switch to go1.20.x at some point yes (go1.21.0 is currently in release-candidate, so go1.19 will reach EOL soon). Best to do that separate though, as updating to new Go releases usually needs a more in-depth review (there may be changes in behavior between versions, some of which can be "subtle" but important).